edx_solutions_api_integration/utils.py

```diff
@@ -401,6 +401,7 @@ def get_profile_image_urls_by_username(username, profile_image_uploaded_at):
             get_profile_image_storage(),
             version=profile_image_uploaded_at.strftime("%s"),
         )
+        urls = {size_display_name: prefix_with_lms_base(url) for size_display_name, url in urls.items()}
```

Comment:
@aamishbaloch this might break things in apros. we should discuss with McKA dev team before prefixing profile image urls with lms domain.

Reply:
@ziafazal should I remove this for now to get other things going?